### PR TITLE
Added cloud cycle ability

### PR DESCRIPTION
**Pull request by JonathanHHenson**
You can cycle through clouds using the arrow keys now.

### CODE REVIEW

File: custom_ui.js [hunk 1]
```diff
@@ -291,6 +291,26 @@ class MaterialSelect extends FormFields {
     }
 
     addSelectorListeners(){
```
Comment (meiamsome) [inline custom_ui.js]: Is this meant to be defined twice?

**Comment by meiamsome** [comment]
It would be great if you could merge in the changes upstream again for me... Thanks

**Comment by JonathanHHenson** [comment]
@meiamsome I got a little confused when trying to fix conflicts, I'll try to get rid of the weird duplicate

**Comment by meiamsome** [comment]
@TurqDevDesign, @AlcaDesign Review, please

**Comment by JonathanHHenson** [comment]
Everything should work without conflicting now

**Comment by JonathanHHenson** [comment]
Just did a test and everything is working great :)

**Comment by meiamsome** [comment]
Wait is there no changes to custom_ui now?

**Comment by meiamsome** [comment]
It looks like it works fine - except that the selection menu's label stays the same.

**Comment by JonathanHHenson** [comment]
@meiamsome, I don't need it anymore now

**Comment by JonathanHHenson** [comment]
Oh, yes

**Comment by JonathanHHenson** [comment]
I forgot to look into that

**Comment by JonathanHHenson** [comment]
It's technically not broken though... :stuck_out_tongue:

**Comment by JonathanHHenson** [comment]
Just setting up my environment, I'm tired of doing code editing on the github website

**Comment by AlcaDesign** [comment]
I suggest we/I/@TurqDevDesign add the ability to set the current option programmatically in `MaterialSelect`. You also need to take into account that I added sorting to the list so just iterating through the index will appear random rather than in the order that appears in the list.

**Comment by JonathanHHenson** [comment]
That would help a tonne!

**Comment by TurqDevDesign** [comment]
I never got an email regarding this thread, I apologize that I haven't acknowledged it. Has anyone began implementing these changes?

**Pull request closed without merging**